Source/Core/Core/FifoPlayer/FifoDataFile.cpp

```diff
@@ -72,6 +72,11 @@ FifoDataFile::FifoDataFile() = default;
 
 FifoDataFile::~FifoDataFile() = default;
 
+bool FifoDataFile::ShouldGenerateFakeVIUpdates() const
+{
+  return m_Version <= 4;
```

Comment:
Don't forget a comment to explain when/why/w.e.-is-appropriate the stub should be changed.